gateway/1.4/modules/ROOT/pages/flex-conn-manage-public-api.adoc

```diff
@@ -5,26 +5,19 @@ endif::[]
 :imagesdir: ../assets/images
 :page-mode: conn
 
-You can manage Flex Gateway in Connected Mode using the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/api-manager-api/[API Manager API] from the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/[Anypoint Platform Developer Portal] instead of using API Manager user interface through Anypoint Platform. This enables you to use the command line to communicate with API Manager.
+You can manage Flex Gateway in Connected Mode using the API Manager API from the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/[Anypoint Platform Developer Portal^] instead of using API Manager user interface through Anypoint Platform. This enables you to use the command line to communicate with API Manager. Using the steps in this tutorial you can create, deploy, apply a policy to, and delete an API instance.
```

Comment:
```suggestion
You can manage Flex Gateway in Connected Mode using the API Manager API from the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/[Anypoint Platform Developer Portal^] instead of using the API Manager user interface through Anypoint Platform. This enables you to use the command line to communicate with API Manager. Using the steps in this tutorial you can create, deploy, apply a policy to, and delete an API instance.
```

Reply:
Regarding the text above, is Anypoint Platform Developer Portal supposed to be initial capped?  (Just checking.)